superset-frontend/src/cccs-viz/plugins/plugin-chart-email-renderer/src/plugin/buildQuery.ts

```diff
@@ -46,7 +46,7 @@ export default function buildQuery(formData: QueryFormData): QueryContext {
   return buildQueryContext(modifiedFormData, (baseQueryObject) => [
     {
       ...baseQueryObject,
-      row_limit: 2,  // Only 1 row is permitted for email rendering
```

Comment:
Sorry, Ryan answered my question in his comment! I see why you had 2 before now and I think that makes the most sense. Better we get the error than rendering the first email in a set of results. Let's put it back to 2. :)